candies/cli/parsers/standard.py

```diff
@@ -62,7 +60,32 @@ def parse(self, args: Iterable[str]) -> Iterable[Invocation]:
         return invocations
 
 
-def configure(parser: ArgumentParser, command: Command, prefix: str = '.'):
+def help(command: Command) -> Type:
+    """Constructs a subclass of an `argparse.HelpFormatter` for a `command`."""
```

Comment:
Is `an` and `a` articles are a good fit here?

Reply:
I don't know, probably not. Will remove them.